docs/porting/target/tickless/tickless.md

```diff
@@ -0,0 +1,64 @@
+# Tickless
+
+## Tickless mode
+
+Mbed OS has SysTick timer which generates tick in a periodic interval. The interrupt generated on every tick to reschedule the threads. The periodic tick configures as 1ms for Mbed OS.
+
+![](./resources/Tick.png)
+
+Tickless mode is an optimization mechanism to suspend the SysTick timer. In particular, this will put the system to uninterrupted sleep to achieve power saving when idle for multiple ticks. Target in tickless mode normally disables the SysTick, sets up wake-up timers and enters sleep mode when idle. When SysTick timer expires on its own or from an external event like interrupt, then it exits sleep mode and re-enables the SysTick.
+
+[Mbed OS sleep modes](../../../apis/power-management-sleep.html):
+
+- Sleep mode: Both low and high-frequency clocks are active.
+- Deep sleep mode: Only the low-frequency clock is active.
+
+To enable tickless mode in Mbed OS, your target has to meet following requirements:
```

Comment:
Before this section you should explain the issue with using two clocks, system timer and low power timer. This causes drift. To avoid it, Mbed OS overrides SysTick with the low power timer.

Reply:
You could then say that it is also possible to enable tickless with the microsecond ticker only. This avoids the drift issue (one source clock only) but prevents deep sleep.